docs/setup/metal3-setup.md

```diff
@@ -55,11 +55,12 @@ Command-line flags may be passed to Ansible via the script, for example to disab
   ./02_configure_host.sh -vvv -e "libvirt_network_dhcp=false"
   ```
 
-Note this configures the environment to deploy downstream clusters with openSUSE Leap, for SLEMicro follow the additional steps below.
+Note this configures the environment to deploy management and downstream clusters with [openSUSE Leap Micro](https://get.opensuse.org/leapmicro/5.5/),
```

Comment:
Shall we put the versions just in case?
```suggestion
Note this configures the environment to deploy management and downstream clusters with [openSUSE Leap Micro 5.5](https://get.opensuse.org/leapmicro/5.5/),
```

Reply:
Thanks, added!